recipes/pyaedt/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "pyaedt" %}
+{% set version = "0.6.28" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyaedt-{{ version }}.tar.gz
+  sha256: 1fd0c485ecdf43fb506f79964c559b86d9ca908a4272fc11a17c5efb82c75a01
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - mono  # [linux]
+    - dotnet-aspnetcore ==3.1.23  # [linux]
+
+  host:
+    - pip
+    - python
+    - dotnet-aspnetcore ==3.1.23  # [linux]
+    - mono  # [linux]
+  run:
+    - python
+    - mono  # [linux]
+    - dotnet-aspnetcore ==3.1.23  # [linux]
+    - cffi ==1.15.0  # [linux]
+    - pywin32 >=303  # [win]
+    - pythonnet ==3.0.0.post1
+    - rpyc ==5.0.1
+    - pyvista >=0.34.1
+    - numpy
+    - matplotlib-base
+    - psutil
+    - pandas
+
+test:
+  imports:
+    - pyaedt
+  requires:
+    - pip
+
```

Comment:
Unless you intend to run `pip check` to test (which it's not right now) you don't need this.

Reply:
> Just a few minor things. Also need @maxcapodi78 to confirm they're willing to be listed as a maintainer for this recipe.

it's ok for me to be listed in maintainers

Reply:
@dopplershift Great catch, I did performed some pip check "test" originally. Thanks.